hypershift-operator/main.go

```diff
@@ -159,6 +160,7 @@ func run(ctx context.Context, opts *StartOptions, log logr.Logger) error {
 		return fmt.Errorf("failed to find operator image: %w", err)
 	}
 	log.Info("using hosted control plane operator image", "operator-image", operatorImage)
+	version.HyperShiftImage = operatorImage
```

Comment:
Is it ok to use this global to store the image?

Reply:
I'd prefer to add it as a pseudo-component image here: https://github.com/openshift/hypershift/blob/b1b263a5aa9edb19d66f2658687741dd2ddd534b/control-plane-operator/main.go#L151-L154
in the same way we do it for other images